orchestration/src/main/java/com/sap/ai/sdk/orchestration/OrchestrationClient.java

```diff
@@ -144,6 +147,54 @@ public CompletionPostResponse executeRequest(@Nonnull final CompletionPostReques
     return executeRequest(postRequest);
   }
 
+  /**
+   * Perform a request to the orchestration service using a module configuration provided as JSON
+   * string. This can be useful when building a configuration in the AI Launchpad UI and exporting
+   * it as JSON. Furthermore, this allows for using features that are not yet supported natively by
+   * the API.
+   *
+   * <p><b>NOTE:</b> This method does not support streaming.
+   *
+   * @param prompt The input parameters and optionally message history to use for prompt execution.
+   * @param moduleConfig The module configuration in JSON format.
+   * @return The completion response.
+   * @throws OrchestrationClientException If the request fails.
+   */
+  @Beta
+  @Nonnull
+  public OrchestrationChatResponse executeRequestFromJsonModuleConfig(
+      @Nonnull final OrchestrationPrompt prompt, @Nonnull final String moduleConfig)
```

Comment:
```suggestion
  public OrchestrationChatResponse executeJsonRequest(
      @Nonnull final OrchestrationPrompt prompt, @Nonnull final String json)
```
It's not called a module config in AI Launchpad right? It's only called "JSON"

Reply:
I didn't name it that way because we don't expect the full request JSON. In Launchpad it's called `Orchestration workflow configuration`